### PR TITLE
Add function to delete a workflow run by ID

### PR DESCRIPTION
**Pull request by xabi93**
This Pull request Adds a new function to delete workflows run from workflow history. The same functionality as this button:
Screenshot 2021-11-22 at 11 43 41

I cannot reference the documentation in the function comment because is missing the endpoint in the docs

### CODE REVIEW

File: github/actions_workflow_runs.go
Comment (gmlewis) [inline github/actions_workflow_runs.go:214]:
```suggestion
// DeleteWorkflowRun deletes a workflow run by ID.
//
// GitHub API docs: https://docs.github.com/en/rest/reference/actions#delete-a-workflow-run
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @xabi93 !
Just one tweak, please, then we will be ready for a second LGTM before merging.

**Comment by xabi93** [comment]
@gmlewis done!! 😅

**Review by gmlewis — APPROVED** [review]
Thank you, @xabi93 !
LGTM.

Awaiting second LGTM/Approval from any other contributor to this repo (we don't have to wait for anyone in particular... all contributors are welcome to participate).

**Review by wesleimp — APPROVED** [review]
LGTM 👌🏼

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
